tasks/modify_settings.yml

```
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Configure setting with value
+  when: item.previous | d() != 'replaced'
+  vars:
+    __bootloader_with_value: >-
+      {{ item.name is defined and item.value is defined }}
+    __bootloader_setting: >-
+      {{ __bootloader_with_value | ternary(item.name + '=' + item.value | d(),
```

```suggestion
      {{ __bootloader_with_value | ternary(item.name + '=' + item.value,
```
if `__bootloader_with_value` is `true`, then both `item.name` and `item.value` must be defined.  adding `| d()` implies that something else is possible.

---

Ansible argues that item.value is not defined when I remove ` | d()`.
```
TASK [linux-system-roles.bootloader : Configure boot setting {{ __bootloader_setting }}] ************************************************************************************
fatal: [127.0.0.3]: FAILED! => {"msg": "The conditional check '((__bootloader_setting not in __bootloader_check_setting.stdout) | bool) == ((not __bootloader_absent) | bool)' failed. The error was: error while evaluating conditional (((__bootloader_setting not in __bootloader_check_setting.stdout) | bool) == ((not __bootloader_absent) | bool)): {{ __bootloader_with_value | ternary(item.name | string + '=' + item.value | string, item.name) }}: 'dict object' has no attribute 'value'\n\nThe error appears to be in '/home/spetrosi/Documents/work/linux-system-roles/bootloader/tests/roles/linux-system-roles.bootloader/tasks/modify_settings.yml': line 32, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n    - name: Configure boot setting {{ __bootloader_setting }}\n      ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}

```